packages/cli/src/commands/service/execute.ts

```diff
@@ -78,21 +88,47 @@ export default class Execute extends Command {
             }), {}))
         }
       },
+      {
+        title: 'Listening for result',
+        task: async () => {
+          const payload: StreamRequest = {
+            filter: {
+              executorHash: runner.hash,
+              instanceHash: runner.instanceHash,
+              tags: [id],
+              taskKey: args.TASK,
+              statuses: [Status.Completed, Status.Failed]
+            }
+          }
+          logs = await this.orchestrator.execution.stream(payload, sign(payload, config.mnemonic))
+        }
+      },
       {
         title: 'Executing task',
         task: async () => {
-          execution = await this.app.executeTaskAndWaitResult({
-            eventHash: base58.decode(flags.eventHash || '6aUPZhmnFKiSsHXRaddbnqsKKi9KogbQNiKUcpivaohb'),
-            executorHash: base58.decode(runner.hash),
-            inputs,
-            tags: ['CLI'],
-            taskKey: args.TASK
+          const payload: CreateRequest = {
+            executorHash: runner.hash,
+            inputs: inputs,
+            taskKey: args.TASK,
+            price: '10000atto',
+            tags: [id]
+          }
+          await this.orchestrator.execution.create(
+            payload,
+            sign({ ...payload, inputs: toStruct(payload.inputs) }, config.mnemonic)
+          )
+          return new Promise((resolve, reject) => {
+            logs.once('data', (exec: IExecution) => {
+              execution = exec
+              execution.status === Status.Completed ? resolve(execution) : reject(new Error(execution.error))
+            })
           })
         }
       }
     ])
     await tasks.run()
 
-    cli.styledJSON(decode(execution.outputs))
+    styledJSON(decode(execution.outputs))
+    process.exit(0)
```

Comment:
This is to exit the process without having to cancel the log and have the warning from GRPC.

Reply:
it will be nicer to catch the error and not displaying it in order to really remove the stream from the server.
But ok as it's local dev only